src/main/java/seedu/address/model/room/exceptions/DuplicateRoomException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.room.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Rooms (Rooms are considered duplicates if they
+ * have the same identity).
+ */
+public class DuplicateRoomException extends RuntimeException {
```

Comment:
Does `DuplicateRoom` apply only to identical `RoomNumber`?

Reply:
yes